labels/labels.go

```diff
@@ -0,0 +1,13 @@
+package labels
+
+const (
+	// label names.
+	ContainerlabLabel = "containerlab"
+	NodeNameLabel     = "clab-node-name"
+	NodeKindLabel     = "clab-node-kind"
+	NodeTypeLabel     = "clab-node-type"
+	NodeGroupLabel    = "clab-node-group"
+	NodeLabDirLabel   = "clab-node-lab-dir"
+	TopoFileLabel     = "clab-topo-file"
```

Comment:
It might be better to remove `Label` suffix from these constants. They look somewhat redundant as the package name is `labels` and they are not consistent with `NodeMgmtNetBr` that doesn't have `Label` suffix.

Reply:
I agree, did that in [859f22b](https://github.com/srl-labs/containerlab/pull/1154/commits/859f22be0f93696ff3542dd7b8f2e2135b7934e8)